pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -46,6 +46,8 @@ type OperatorVersionSpec struct {
 
 	// UpgradableFrom lists all OperatorVersions that can upgrade to this OperatorVersion.
 	UpgradableFrom []OperatorVersion `json:"upgradableFrom,omitempty"`
+
+	CrdVersion string `json:"crdVersion,omitempty"`
```

Comment:
huh? What is CrdVersion? Is that probably something we should rather remove from CRD than add in here?

Reply:
Do you think we could remove it from the CRD without having to bump to `v1beta2`?

Reply:
I personally thing it's a safe thing to do if we make sure that the field was not used in 0.8.0 and 0.9.0 - before that, we did not provide backward compatibility anyway so it does not matter (but I doubt it was used ever...). I don't even know what would be the purpose of this to be honest.

Reply:
I looked at the history (`git log -p`) all the way to the very beginning and it seems it was never used.
Reverted that bit here, and will send another PR to remove it from the structs.
PTAL @alenkacz

Reply:
Created PR https://github.com/kudobuilder/kudo/pull/1204